src/main/java/com/google/maps/OkHttpRequestHandler.java

```diff
@@ -40,6 +44,7 @@
 
   private final OkHttpClient client = new OkHttpClient();
   private final RateLimitExecutorService rateLimitExecutorService;
+  public static final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
```

Comment:
please make private, and move it up with the other private static final members.